go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/filecoin-project/go-address v0.0.0-20200107215422-da8eea2842b5
+	github.com/filecoin-project/go-crypto v0.0.0-20191218222705-effae4ea9f03
```

Comment:
used here: https://github.com/filecoin-project/specs-actors/pull/56/files#diff-d5777d603b48d25386edace2468db7cfR244-R256

Reply:
Revert after removing that use.